python/requests/best-practice/use-response-json-shortcut.yaml

```diff
@@ -0,0 +1,17 @@
+rules:
+- id: python.requests.best-practice.use-response-json-shortcut
+  patterns:
+  - pattern-inside: import json; ...
+  - pattern-inside: import requests; ...
+  - pattern-inside: $RESP = requests.$METHOD(...); ...
```

Comment:
Not sure if in practice code will always be in the form of this assignment: `$RESP = requests.$METHOD(...);` I'm trying to think if the `requests` call could ever be in a loop or conditional or used in some other way this might (incorrectly) miss (false negative).

Probably fine for now, just wanted to mention it.

Reply:
it could technically be used with the `:=` operator 🤔